Makefile

```diff
@@ -37,6 +37,11 @@ csv:
 ddsidl:
 	${TOOLSDIR}/vspec2ddsidl.py -I ./spec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).idl
 
+# Verifies that supported overlay combinations are syntactically correct.
+overlays:
```

Comment:
Only indirectly related to the contribution - make sure that the overlay is syntactically correct